index.html

```diff
@@ -459,6 +462,164 @@
             behavior for any source supported by <a>HTMLMediaElement</a>.
           </div>
         </section>
+        <section>
+          <h4>The <dfn for="RemotePlayback"><code>state</code></dfn> attribute</h4>
+          <p>
+            The <dfn><code>state</code></dfn> attribute represents the
+            <a>remote playback</a> connection's current state. It can take one of
+            the values of <a>RemotePlaybackState</a> depending on the
+            connection state:
+          </p>
+          <ul dfn-for="RemotePlaybackState">
+            <li>
+              <dfn>connecting</dfn> means that the user agent is attempting to
+              <a>initiate remote playback</a> with the selected <a>remote playback device</a>.
+              This is the initial state when the <code>promise</code> is
+              resolved with <code>true</code> by <code><a for="RemotePlayback">connect</a>()</code> method.
```

Comment:
rephrase: This is the initial state when the promise returned by connect() is resolved with true.